ai-decision-services/ai-ad-dpp-engine/setup-ad/setup-anomaly-detection.md

```diff
@@ -0,0 +1,10 @@
+Lab 1: Setup Anomaly Detection
+=================
+
+Here we will use AD as the example of post processing. Go to AI Services → Anomaly Detection, 
+
+![](./images/Set-AD1.png)
+and create a project there.
```

Comment:
@bug-catcher.  Overall, every Lab's markup/content requires a bit of polishing.
Reword to -> First, create a project.